### PR TITLE
FORNO-659 Add overwriteExisting flag to media-import cmd

### PR DESCRIPTION
**Pull request by aswasif007**
## Ticket
[FORNO-659 | Media Import: Add flag to force overwrite of existing files to CLI](https://vipjira.atlassian.net/browse/FORNO-659)

## PR Dependencies
https://github.com/Automattic/vip-go-api-public/pull/1115

## Description
We are adding `overwriteExisting` or `-o` flag to the `vip import-media` command. This will send `overwriteExistingFiles: true` to Parker's `start-media-import` mutation.

## How to test
_(WIP)_
1. Pull the PR.
2. Pull the latest master of GOOP.
3. Start GOOP using `docker-compose up` in GOOP repo.
3. Pull and checkout the dependency PR branch in Parker.
3. Start Parker using `docker-compose up` in Parker repo.
4. Create a authorization token using `./bin/generate-token.sh` in Parker.
5. Change [this](https://github.com/Automattic/vip/blob/master/src/lib/api.js#L35) line to make it look like the following:
```javascript
Authorization: 'Bearer <Generated Token>`
``` 
8. Run `npm run build`
9. Run the following command:
```bash
VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip-import-media.js @1000.production test.zip -o
```

### CODE REVIEW

**Comment by vincenzo** [comment]
Code looks good. I'll come around tomorrow for testing.

**Comment by vincenzo** [comment]
I keep getting `App does not exist` for any app ID. Not sure why. I'll get back to it later.

File: src/bin/vip-import-media.js [hunk 1]
```diff
@@ -94,9 +94,10 @@ Are you sure you want to import the contents of the url?
 } )
 	.command( 'status', 'Check the status of the current running import' )
 	.option( 'exportFileErrorsToJson', 'Export any file errors encountered to a JSON file instead of a plain text file', false )
+	.option( 'overwriteExisting', 'Overwrite any existing files', false )
```
Comment (saroshaga) [inline src/bin/vip-import-media.js]: I'd recommend keeping Parker and CLI in sync. So `overwriteExistingFiles`

**Review by saroshaga — APPROVED** [review]
Left a suggestion but it looks good otherwise 🚀

File: src/bin/vip-import-media.js (hunk 1 above)
Comment (aswasif007) [reply src/bin/vip-import-media.js]: Addressed here: https://github.com/Automattic/vip/pull/739/commits/f0a4738c5c1dd4989a827dae951d111394e84fec

**Comment by saroshaga** [comment]
@aswasif007 : any way to test this end to end?

**Comment by aswasif007** [comment]
@saroshaga I couldn't figure out a way. Perhaps it would work if we can ensure that the cli calls Parker with the mutation mentioned in this PR: https://github.com/Automattic/vip-go-api-public/pull/1115

**Pull request merged**